### DISCUSSION TITLE
Different captcha methods

### DISCUSSION THREAD

**Discussion by TechStreetDev** [pelican-dev/panel#51] [Ideas]
Allow options to choose from different captchas, like Google reCAPCHA, Cloudflare Turnstile, hCapcha and any others which make sense. 

This allows people to use more privacy focused captchas like cloudflare turnstile.

**Comment by HughParry** [comment]
Cloudflare only manages to be privacy preserving by blocking more privacy focussed readers! https://www.oopspam.com/blog/cloudflare-turnstile

Check out https://prosopo.io/ if you want a captcha that's privacy preserving but doesn't block people using VPNs

**Comment by lancepioch** [comment]
Thanks, you will be able to install plugins of your choice that does this!
